libsolidity/CompilerContext.h

```diff
@@ -115,9 +115,12 @@ class CompilerContext
 	u256 m_stateVariablesSize = 0;
 	/// Storage offsets of state variables
 	std::map<Declaration const*, u256> m_stateVariables;
-	/// Positions of local variables on the stack.
+	/// Offsets of local variables on the stack (relative to stack base).
 	std::map<Declaration const*, unsigned> m_localVariables;
-	/// Labels pointing to the entry points of funcitons.
+	/// Sum of stack sizes of local variables
+	unsigned m_localVariablesSize;
```

Comment:
Is this used somewhere? Did you add this or did it result from a merge conflict?
There is also a blank line below.